management/server/rego/default_policy.rego

```diff
@@ -0,0 +1,10 @@
+package netbird
+
+# all rules for peer connectivity and firewall
```

Comment:
This comment is saved to the store and duplicated for each policy. Let's remove it